doc/source/data/api/from_other_data_libs.rst

```diff
@@ -0,0 +1,88 @@
+.. _api-guide-for-users-from-other-data-libs:
+
+API Guide for Users from Other Data Libraries
+=============================================
+
+Ray Datasets is a data loading and preprocessing library for ML. It shares certain
+similarities with other ETL data processing libraries, but also has its own focus.
+In this API guide, we will provide API mappings for users who come from those data
+libraries, so you can quickly map what you may already know to Ray Datasets APIs.
+
+Note:
+
+- This is meant to map APIs that perform comparable but not necessarily identical operations.
+  Please check the API reference for exact semantics and usage.
+- This list may not be exhaustive: Ray Datasets is not a traditional ETL data processing library, so not all data processing APIs can map to Datasets.
+  In addition, we try to focus on common APIs or APIs that are less obvious to see a connection.
+
+.. _api-guide-for-pandas-users:
+
+For Pandas Users
+----------------
+
+.. list-table:: Pandas DataFrame vs. Ray Datasets APIs
+   :header-rows: 1
+
+   * - Pandas DataFrame API
+     - Ray Datasets API
+   * - pandas.DataFrame.head()
```

Comment:
It would be nice if each of these Pandas APIs linked out the corresponding API docs page for the latest stable Pandas release!

Reply:
Nit: You could shorten these to e.g. `df.head()`, where the `df.` prefix would probably be more readable than the `pandas.DataFrame.` prefix, and better matches the Datasets side with the `ds.` prefix.